jetbrains-core/src/software/aws/toolkits/jetbrains/services/cloudwatch/logs/insights/AddRemoveLogGroupTable.kt

```diff
@@ -0,0 +1,46 @@
+// Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package software.aws.toolkits.jetbrains.services.cloudwatch.logs.insights
+import com.intellij.execution.util.ListTableWithButtons
+import com.intellij.openapi.project.Project
+import com.intellij.util.ui.ListTableModel
+import software.aws.toolkits.resources.message
+import javax.swing.table.TableCellEditor
+
+class AddRemoveLogGroupTable(project: Project) : AddRemoveLogs<SelectedLogGroups> (emptyTableMainText = "No log groups found",
+    addNewEntryText = "Add log groups") {
+    override fun cloneElement(variable: SelectedLogGroups): SelectedLogGroups = variable.copy()
+    override fun createElement(): SelectedLogGroups = SelectedLogGroups()
```

Comment:
why is `createElement` overwritten?

Reply:
Displaying a sample table. Added the TODO section in the class

Reply:
You can do this in `init`, would prefer

Reply:
Added the TODO section to init